wled00/wled_serial.cpp

```diff
@@ -19,6 +19,15 @@ enum class AdaState {
   TPM2_Header_CountLo,
 };
 
+void update_baud_rate(int rate){
+  if (!pinManager.isPinAllocated(1)){
```

Comment:
Please add proper pin allocation detection.
isPinAllocated() supports PinOwner class and utilize that for detection (in this case it is allowed to use GPIO1 when PinOwner is DebugOut).

Reply:
I will go back and take a look.

I used isPinAllocated(1) here because it is used elseware already in wled_serial for returning state and info JSON.
https://github.com/Aircoookie/WLED/blob/master/wled00/wled_serial.cpp#L63-L64